dime-coding-standards/checklists/tex/Reviewer Feedback Checklist

```diff
@@ -0,0 +1,154 @@
+{}% before the title body
+[]% after the title body
+
+\newcommand{\dimeCheckBox}[1]{\CheckBox[height=0.01cm, width=0.4cm, bordercolor=gray]{#1}}
+\newcommand{\dimeTextField}[3]{\TextField[name=#1, height=0.3cm, width=#2, bordercolor=gray]{#3}}
+
+
+\newcommand{\titleBox}[1]{
+	\begin{tcolorbox}
+		[colframe = fontcolor,
+		colback = fontcolor,
+		sharp corners,
+		halign = flush center,
+		valign = center,
+		height = 0.3\textwidth,
+		after skip = 1cm]
+		#1
+	\end{tcolorbox}
+}
+
+\begin{document}
+	\begin{fullwidth}
+		
+		\titleBox{
+			\textcolor{white}{\LARGE{\textbf{DIME Analytics \\ Peer Code Review - Reviewer Feedback Checklist}} \\
+				\Large\textbf{{v1.0}}}
+		}
+
+        \dimeTextField{time}{2.5 cm}{Reviewer:} 
+
+        \dimeTextField{time}{2.5 cm} {Coder:}
+        
+        
+        This checklist lists important factors to consider while reviewing your code review partner's code package. Please fill this checklist, and submit it as an attachment when you submit \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this detailed form.}
+        
+		\section*{Overview - First Impression}
+
+        Once you have confirmed that you received a package from your code review partner, please refer to the following list for basic recommended practices:
+        \begin{itemize}
+			\setlength\itemsep{-0.1em}		
+			
+			\item[] \dimeCheckBox{ Scripts for data cleaning, variable creation, and analysis are separate; and analysis scripts do not include any data processing, unless necessary for the creation of a table or graphic}
+			\item[] \dimeCheckBox{ Variable construction scripts include detailed comments about each variable created}
+			\item[] \dimeCheckBox{ Analysis scripts do not depend on having the results of other scripts in memory, except for the Master script}
+			\item[] \dimeCheckBox{ All scripts are well-commented and formatted, such that one can easily identify functional chunks of code and evaluate whether they correctly implement the econometric or statistical process described}
+			\item[] \dimeCheckBox{ Tables are output as .csv or .tex files or other raw text files when possible}
+		\end{itemize}
+	
+		\vspace{.6em}
+
+        \section*{Reproducibility}
+
+        \begin{itemize}
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{Code ran perfectly on the first run \textbf{(Applicable only if data was shared with you.)}}
+			\item[] \dimeCheckBox{Outputs shared (if any) replicated perfectly on the first run}
+            \item[] \dimeCheckBox{If not, was able to determine the reason for why outputs changed}
+			\item[] \dimeCheckBox{Make note of any issues you faced in these steps. You will need to provide this information in \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this detailed review summary form.}}
+			
+		\end{itemize}
+		
+        \vspace{.6em}
+
+       \section*{Ease of Use and Understanding}
```

Comment:
add intro note 

"Next, assess how easy it is to understand the code you are reviewing. Consider whether the documentation provided is sufficient, and whether the code is organized in such a way that it would be easy for someone else to work on it. Make note of what additional documentation is needed as you will be asked to share detailed feedback in the submission review.